JoinsLab/app/src/main/java/com/justinwells/joinslab/Helper.java

```diff
@@ -0,0 +1,158 @@
+
+    private Helper (Context context) {
+        super(context,"db",null,1);
+    }
+
+    public static Helper getInstance (Context context) {
+        if (sInstance == null) {
+            sInstance = new Helper(context);
+        }
+        return sInstance;
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase sqLiteDatabase) {
+        sqLiteDatabase.execSQL(CREATE_TABLE_EMPLOYEES);
+        sqLiteDatabase.execSQL(CREATE_TABLE_JOBS);
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {
+        sqLiteDatabase.execSQL("DROP TABLE IF EXISTS " + EMPLOYEE_TABLE_NAME);
+        sqLiteDatabase.execSQL("DROP TABLE IF EXISTS " + JOB_TABLE_NAME);
+        onCreate(sqLiteDatabase);
+    }
+
+    public void insertRowEmployee (Employee employee) {
+        SQLiteDatabase db = getWritableDatabase();
+        ContentValues values = new ContentValues();
+        values.put(COL_SSN, employee.getSsn());
+        values.put(COL_FIRST, employee.getFirstName());
+        values.put(COL_LAST, employee.getLastName());
+        values.put(COL_YEAR, employee.getBirthYear());
+        values.put(COL_CITY, employee.getCity());
+        db.insertOrThrow(EMPLOYEE_TABLE_NAME,null,values);
+        db.close();
+    }
+
+    public void insertRowJobs (Job job) {
+        SQLiteDatabase db = getWritableDatabase();
+        ContentValues values = new ContentValues();
+        values.put(COL_SSN, job.getSsn());
+        values.put(COL_COMPANY, job.getCompany());
+        values.put(COL_SALARY, job.getSalary());
+        values.put(COL_EXP, job.getExp());
+        db.insertOrThrow(JOB_TABLE_NAME, null, values);
+        db.close();
+    }
+
+    public List<String> getMacyEmployees () {
+        List<String>results = new ArrayList<>();
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT " + COL_FIRST + ", " + COL_LAST + " FROM " + EMPLOYEE_TABLE_NAME+
+                " INNER JOIN " + JOB_TABLE_NAME + " ON " + EMPLOYEE_TABLE_NAME + "." + COL_SSN +
+                " = " + JOB_TABLE_NAME + "." + COL_SSN + " WHERE " + COL_COMPANY  + " = 'Macys' ";
+
+
+        Cursor cursor = db.rawQuery(query, null);
+
+        if(cursor.moveToFirst()){
+            while(!cursor.isAfterLast()){
+                results.add(cursor.getString(cursor.getColumnIndex(COL_FIRST)) + " " +
+                            cursor.getString(cursor.getColumnIndex(COL_LAST)));
+                cursor.moveToNext();
+            }
+        }
+        cursor.close();
+        return results;
+    }
+
+    public List<String> getHighestPaid () {
+        List<String>highestNames = new ArrayList<>();
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT " + COL_FIRST + COL_LAST + " FROM " + EMPLOYEE_TABLE_NAME +
+                "INNER JOIN " + JOB_TABLE_NAME + " ON " + EMPLOYEE_TABLE_NAME + "." + COL_SSN +
+                " = " + JOB_TABLE_NAME + "." + COL_SSN + " ORDER BY " + COL_SALARY;
```

Comment:
You'll want to add `DESC` at the end of your `ORDER BY` clause. By default, it sorts in _ascending_ order from smallest to largest. In this case, you want _descending_ order, from largest to smallest.